homeassistant/components/daikin/config_flow.py

```diff
@@ -128,7 +128,7 @@ async def async_step_discovery(self, discovery_info):
     async def async_step_zeroconf(self, discovery_info):
         """Prepare configuration for a discovered Daikin device."""
         _LOGGER.debug("Zeroconf discovery_info: %s", discovery_info)
-        await self.async_set_unique_id(discovery_info.get(CONF_HOST))
+        await self.async_set_unique_id(discovery_info.get(KEY_HOSTNAME))
         self._abort_if_unique_id_configured()
-        self.host = discovery_info.get(CONF_HOST)
+        self.host = discovery_info.get(KEY_HOSTNAME)
```

Comment:
I don't think we should change here. The host attribute should be the ip address, right?

Reply:
Host does not exist, it is called hostname in the zeroconf result.

Reply:
What about the result you posted here?
https://github.com/home-assistant/core/pull/35769#discussion_r431796978

Reply:
True, I'll revert that back.

Thanks for spotting it.